tests/test_api.py

```diff
@@ -1952,11 +1956,14 @@ def test_hierarchical_hashes_array_join(self) -> None:
         assert response.status_code == 200
         result = json.loads(response.data)
 
-        assert result["sql"].startswith(
+        val = (
             "SELECT arrayJoin((arrayMap((x -> replaceAll(toString(x), '-', '')), "
-            "hierarchical_hashes) AS _snuba_hierarchical_hashes)) FROM errors_local PREWHERE"
+            "hierarchical_hashes) AS _snuba_hierarchical_hashes)) FROM %s PREWHERE"
+            % (errors_storage.get_table_writer().get_schema().get_table_name())
```

Comment:
Similar to above, how about using an f-string here too?

Reply:
Done!